sympy/printing/tests/test_mathml.py

```diff
@@ -3,7 +3,7 @@
     pi, GoldenRatio, EulerGamma, Sum, Eq, Ne, Ge, Lt, Float, Matrix, Basic, S
 from sympy.printing.mathml import mathml, MathMLContentPrinter, MathMLPresentationPrinter, \
     MathMLPrinter
-
+from sympy import print_mathml
```

Comment:
Remove this as it is not needed. 

(In general it is better to import from the lowest level in tests, to test that the method is where one thinks it is.)